Header header
bool IGNOK
bool IMD
bool PRESSURE
bool BMSDE